### PR TITLE
Fix TextBox keyboard hotkeys operability in IE11 (T944726, T944493)

### PR DESCRIPTION
**Pull request by DokaRus**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js [hunk 1]
```diff
@@ -95,6 +95,26 @@ QUnit.module('common', {}, () => {
         }
     });
 
+    QUnit.test('"maxLength" option on IE should works correctly with the hotkeys (T944726, T944493)', function(assert) {
+        const originalIE = browser.msie;
+
+        try {
+            browser.msie = true;
+            const $element = $('#textbox').dxTextBox({ maxLength: 1, value: 'b' });
+            const $input = $element.find('.' + INPUT_CLASS);
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js]:
```suggestion
            const $input = $element.find(`.${INPUT_CLASS}`);
```

File: testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js [hunk 2]
```diff
@@ -95,6 +95,26 @@ QUnit.module('common', {}, () => {
...
+            let event = $.Event('keydown', { key: 'a', ctrlKey: true });
+
+            $input.trigger(event);
+            assert.ok(!event.isDefaultPrevented(), 'default is not prevented');
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js]:
```suggestion
            assert.notOk(event.isDefaultPrevented(), 'default is not prevented');
```

File: testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js [hunk 3]
```diff
@@ -95,6 +95,26 @@ QUnit.module('common', {}, () => {
...
+
+            event = $.Event('keydown', { key: 'z', ctrlKey: true });
+            $input.trigger(event);
+            assert.ok(!event.isDefaultPrevented(), 'default is not prevented');
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js]:
```suggestion
            assert.notOk(event.isDefaultPrevented(), 'default is not prevented');
```

File: testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js [hunk 4]
```diff
@@ -95,6 +95,26 @@ QUnit.module('common', {}, () => {
...
+        try {
+            browser.msie = true;
+            const $element = $('#textbox').dxTextBox({ maxLength: 1, value: 'b' });
+            const $input = $element.find('.' + INPUT_CLASS);
+            let event = $.Event('keydown', { key: 'a', ctrlKey: true });
+
+            $input.trigger(event);
+            assert.ok(!event.isDefaultPrevented(), 'default is not prevented');
+
+            event = $.Event('keydown', { key: 'z', ctrlKey: true });
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js:110]: Why should we test the same case with another key?

**Review by ksercs — DISMISSED** [review]

**Review by ksercs — APPROVED** [review]

File: testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js (hunk 4 above)
Comment (DokaRus) [reply testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js:110]: We have two different private bugs so I decided to make the test a little more complicated

**Review by San4es — APPROVED** [review]

**Pull request merged**